### PR TITLE
Video player

### PR DESCRIPTION
**Pull request by dominik-git**
You should see full screen video.

### CODE REVIEW

File: src/components/video-player-container/parts/video-player-controls.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
 // @flow
 import React from 'react';
 import './video-player-controls.css';
-import Right from '../../../../public/assets/RW.svg';
-import Left from '../../../../public/assets/FF.svg';
+import Left from '../../../../public/assets/RW.svg';
+import Right from '../../../../public/assets/FF.svg';
```
Comment (varholak-peter) [inline src/components/video-player-container/parts/video-player-controls.js]: change name to Rewind and FastForward

File: src/components/video-player-container/parts/video-player.js [hunk 2]
```diff
@@ -6,6 +6,7 @@ import 'video.js/dist/video-js.css';
 import videojs from 'video.js';
 import VideoPlayerControls from './video-player-controls';
 import './video-player.css';
+import Left from '../../../../public/assets/RW.svg';
```
Comment (varholak-peter) [inline src/components/video-player-container/parts/video-player.js]: as above

File: src/components/video-player-container/parts/video-player-controls.js (hunk 1 above)
Comment (dominik-git) [inline src/components/video-player-container/parts/video-player-controls.js]: fixed

File: src/components/video-player-container/parts/video-player.js (hunk 2 above)
Comment (dominik-git) [inline src/components/video-player-container/parts/video-player.js]: fixed

**Pull request merged**